docs/addons/cert-manager.md

```diff
@@ -0,0 +1,79 @@
+# cert-manager Add-on
+
+This add-on installs [cert-manager](https://github.com/cert-manager/cert-manager).
+
+cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates.
+It can issue certificates from a variety of supported sources, including Let's Encrypt, HashiCorp Vault, and Venafi as well as private PKI.
+It will ensure certificates are valid and up to date, and attempt to renew certificates at a configured time before expiry.
+
+## Usage
+
+```typescript
+
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.CertManagerAddOn()
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+## Configuration Options
+
+- `version`: Version fo the Helm Chart to be used to install cert-manager
+- `installCRDs`: (boolean) To automatically install and manage the CRDs as part of your Helm release,
+- `createNamespace`: (boolean) If you want CDK to create the namespace for you
+- `values`: Arbitrary values to pass to the chart. Refer to the cert-manager [Helm Chart documentation](https://artifacthub.io/packages/helm/cert-manager/cert-manager) for additional details
+

```

Comment:
let's add that it supports standard helm configuration options (e..g chart/repo/version, etc):

https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/docs/addons/index.md#standard-helm-add-on-configuration-options

E.g. https://aws-quickstart.github.io/cdk-eks-blueprints/addons/argo-cd/#:~:text=Supports%20standard%20helm%20configuration%20options.

Reply:
done